src/main/java/module-info.java

```diff
@@ -144,10 +149,9 @@
     requires transitive org.jspecify;
 
     // other libraries
-    requires org.antlr.antlr4.runtime;
     requires org.libreoffice.uno;
-    requires de.saxsys.mvvmfx.validation;
-    requires dd.plist;
```

Comment:
I think, this is an issue of IntelliJ. I had no time to craft a MWE to demonstrate to them that they destory module-info.java

Reply:
You can search all the code using Ctrl+Shift+F and then type in "plist". Then, you should find usages.